test/parallel/test-crypto-cipher-decipher.js

```diff
@@ -139,3 +139,14 @@ testCipher2(Buffer.from('0123456789abcdef'));
   assert.doesNotThrow(() => txt += decipher.final('utf-16le'));
   assert.strictEqual(txt, plaintext, 'decrypted result in utf-16le');
 }
+
+// setAutoPadding/setAuthTag/setAAD should return this
```

Comment:
Can I suggest that you enclose this in backticks or something? Otherwise the “this” reads like a demonstrative pronoun in this sentence